site/pages/docs/versions/dwnld-en.hbs

```diff
@@ -25,13 +25,13 @@
 		<tbody>
 			<tr>
 				<td>Web Experience Toolkit (WET)</td>
-				<td>v4.0.43.2</td>
-				<td class="text-center"><a class="btn btn-default" href="https://github.com/wet-boew/wet-boew/releases/latest">Download<span class="wb-inv"> - v4.0.43.2 - Web Experience Toolkit (WET) theme</span></a></td>
+				<td>v4.0.44 (STR)</td>
+				<td class="text-center"><a class="btn btn-default" href="https://github.com/wet-boew/wet-boew/releases/latest">Download<span class="wb-inv"> - v4.0.44 - Web Experience Toolkit (WET) theme</span></a></td>
 			</tr>
 			<tr>
 				<td>Canada.ca (GCWeb)</td>
-				<td>v9.5.0</td>
-				<td class="text-center"><a class="btn btn-default" href="https://github.com/wet-boew/GCWeb/releases/latest">Download<span class="wb-inv"> - v9.5.0 - Canada.ca theme</span></a></td>
+				<td>v9.6.0 (STR)</td>
+				<td class="text-center"><a class="btn btn-default" href="https://github.com/wet-boew/GCWeb/releases/latest">Download<span class="wb-inv"> - v9.6.0 - Canada.ca theme</span></a></td>
```

Comment:
```suggestion
				<td class="text-center"><a class="btn btn-default" href="https://github.com/wet-boew/GCWeb/releases/latest">Download<span class="wb-inv"> - v9.6.0 (STR) - Canada.ca theme</span></a></td>
```

Reply:
I made the change and changed the GCWeb version for 10.0.0